src/Lib/Configuration.php

```diff
@@ -455,6 +461,37 @@ public function setJsonOptions(int $jsonOptions)
         return $this;
     }
 
+    /**
+     * @return string
+     */
+    public function getConnectionName(): string
+    {
+        return $this->connectionName;
+    }
+
+    /**
+     * @param string $connectionName Connection name to use when loading tables for building schemas from models.
+     * @return $this
+     */
+    public function setConnectionName(string $connectionName)
+    {
+        $configuredConnections = ConnectionManager::configured();
+
+        if (!in_array($connectionName, $configuredConnections)) {
+            throw new InvalidArgumentException(
```

Comment:
Can we add a test (if one doesn't exist) to make sure this exception is thrown. I don't really care to test anything else beyond this.

Reply:
Already there, I added it to the data provider for the existing ConfigurationTest.

For the rest, noted. I will remove the other test I wrote and the fixture that I created for it.

Reply:
Great, I'll take another look later tonight :-)